src/backend/impress/settings.py

```diff
@@ -849,6 +849,7 @@ class Test(Base):
         "django.contrib.auth.hashers.MD5PasswordHasher",
     ]
     USE_SWAGGER = True
+    STATIC_ROOT = None
```

Comment:
Why do you need this?

Reply:
We have warning in the tests saying that the `/data/static` directory does not exists. We can create it or set to None the `STATIC_ROOT` variable as static are not used in the tests.

Reply:
```
core/tests/test_api_config.py::test_api_config_with_invalid_json_theme_customization_file[False]
core/tests/test_api_config.py::test_api_config_with_invalid_json_theme_customization_file[True]
core/tests/test_api_config.py::test_api_config_with_theme_customization[False]
core/tests/test_api_config.py::test_api_config_with_theme_customization[True]
  /home/runner/.local/lib/python3.13/site-packages/django/core/handlers/base.py:61: UserWarning: No directory at: /data/static/
    mw_instance = middleware(adapted_handler)
```